pkg/ctrl/server/cgroup.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package server
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/chaos-mesh/chaos-mesh/pkg/bpm"
+	"github.com/chaos-mesh/chaos-mesh/pkg/ctrl/server/model"
+)
+
+// GetCgroups returns result of cat /proc/cgroups
+func (r *Resolver) GetCgroups(ctx context.Context, obj *model.PodStressChaos) (*model.Cgroups, error) {
+	cmd := "cat /proc/cgroups"
+	raw, err := r.ExecBypass(ctx, obj.Pod, cmd, bpm.PidNS, bpm.MountNS)
+	if err != nil {
+		return nil, err
+	}
+
+	cgroups := &model.Cgroups{
+		Raw: raw,
+	}
+
+	// no more info for StressngStressors
+	if obj.StressChaos.Spec.StressngStressors != "" || obj.StressChaos.Spec.Stressors == nil {
+		return cgroups, nil
+	}
+
+	isCPU := true
+	if obj.StressChaos.Spec.Stressors.CPUStressor == nil {
+		isCPU = false
+	}
+
+	if isCPU {
+		var cpuMountType string
+		if regexp.MustCompile("(cpu,cpuacct)").MatchString(string(raw)) {
+			cpuMountType = "cpu,cpuacct"
+		} else {
+			cpuMountType = "cpu"
```

Comment:
what is the difference between these two types?

Reply:
The latter means cgroup does not support cpuacct sub-system, I left comments

Reply:
Comments is added.